robot_modeling/swerve/swerve_ctrl.py

```diff
@@ -0,0 +1,173 @@
+
+import os
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from control import ss, tf, LinearIOSystem, NonlinearIOSystem, summing_junction, interconnect, input_output_response
+from lmfit import Parameters
+
+from common.nonlinear_systems import second_order_sys_x2_deadband
+from common.control_utils import clamp, clamp_update
+
+# Module name used by main.py as a user argument.
+SWERVE_MODULE_CTRL = "swerve_ctrl"
+
+# Swerve module dynamics
+SWERVE_MODEL = second_order_sys_x2_deadband
+
+# Swerve module constants
+params = {
```

Comment:
what'd you calculate these from?

Reply:
I got these from the system ID results

Reply:
but the code is used to run the system ID, right?